src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -135,6 +135,24 @@ void Instance::Shutdown()
     mpBaseDriver->Shutdown();
 }
 
+#if !CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION
```

Comment:
Is CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION defined for devices that don't do BLE at all?  I guess probably....

Reply:
Yes, the default is is defined in src/include/platform/CHIPDeviceConfig.h

Reply:
No, what I meant is: is it going to be true for non-BLE devices?  But I am guessing yes.